types/stylelint/index.d.ts

```diff
@@ -681,6 +681,10 @@ declare namespace stylelint {
 		| 'text-emphasis'
 		| 'transition';
 
+	type ImplicitResetters = Extract<ShorthandProperties, 'border' | 'flex' | 'font'>;
```

Comment:
`flex` used to be an implicit resetter.
I chose to not support it for now because it's unclear what's preferable as a default.
i.e. a separate issue needs to be created

Reply:
[suggest] It seems that adding the `ImplicitResetters` and `ShorthandToResetToInitialProperty` types is unnecessary for now because they're not required to be published, not `LonghandSubPropertiesOfShorthandProperties`.

So, how about removing the types?